.github/workflows/ci-lint.yml

```diff
@@ -42,6 +42,11 @@ jobs:
           python -m pip install \
             https://github.com/oasislabs/towncrier/archive/oasis-master.tar.gz \
             gitlint
+      - name: Lint git commits
```

Comment:
Hmm... changing the order will have unintended consequences, i.e. the next step will not be run when _Lint git commits_ step fails...

Reply:
^ I'm willing to make that compromise for the correctness of this step

Reply:
nvm, backing this out due to fixes elsewhere that make its unnecessary

Reply:
This should no longer be necessary since a different fix landed in #2712.